### DISCUSSION TITLE
Error when Import Chem

### DISCUSSION THREAD

**Discussion by Randi-f** [rdkit/rdkit#5262] [Q&A]
I successfully install RDKit and importing rdkit is okay. However, when I try to import Chem, an error is sent back:  from rdkit.Chem.rdmolfiles import *
ImportError: DLL load failed while importing rdmolfiles: 找不到指定的模块。
(window 10, python 3.8) Can anyone please help me with this stuff? ( I also tried to re-download it, but similar error exists)

**Comment by Crispae** [comment]
Did you try to reinstall using "pip install rdkit-pypi"?

**Reply by Randi-f** [reply]
Yeap! And I still had the same error. Can there be any other possible reasons for this? Thank you so much !

**Reply by viktor-ktorvi** [reply]
Hey, did you solve it?

**Reply by lsy586** [reply]
Hello, did you slove it?

**Comment by huangliang0828** [comment]
(window 10-64bit, python 3.9.12) 
I also meet the same problem，using the comand：conda install -c conda-forge rdkit==XXX
 XXX==2022.03.3  or 2021.09.1 ,and other versons

**Reply by viktor-ktorvi** [reply]
Hey, did you solve it?

**Reply by lsy586** [reply]
Hello, did you slove it?
